binrz/man/rizin.1

```diff
@@ -78,84 +82,87 @@ List supported IO plugins
 Map file at given address
 .It Fl M
 Disable demangling
-.It Fl n, nn
+.It Fl n , nn
 Control loading of RzBin info, with -nn only loading binary structures
 .It Fl N
-Disable user settings/projects from ~/.rizinrc, ~/.config/rizin/rizinrc and the scripts inside .../rizinrc.d/ directory
+Disable user settings from ~/.config/rizin/rizinrc. Check RZ_CONFIGHOME in rizin -H output.
```

Comment:
~/.rizinrc is actually valid, but maybe we can just replace the sentence with: Disables loading user settting from the rizinrc file (check the RZ_CONFIGHOME value in the rizin -H output).